### PR TITLE
Adds crop package demo under Misc menu.

### PR DESCRIPTION
**Pull request by xclud**
I tried to add the demo of the package crop to the main application. I was not able to debug the application because of some firebase related initialization and gradle plugin requirements. But i hope it does work correctly.

### CODE REVIEW

File: lib/src/pages/misc/crop.dart [hunk 1]
```diff
@@ -0,0 +1,227 @@
+import 'dart:math';
```
Comment (lohanidamodar) [inline lib/src/pages/misc/crop.dart]: At the top of the demo file, please keep author info in comments, copy format from other demos, use your info.

File: pubspec.yaml [hunk 2]
```diff
@@ -42,6 +42,7 @@ dependencies:
   firebase_core: ^0.4.3+1
   firebase_analytics: ^5.0.9
   flutter_staggered_grid_view: ^0.3.0
+  crop: any
```
Comment (lohanidamodar) [inline pubspec.yaml]: Please use fixed / latest version of the package

Comment (xclud) [reply pubspec.yaml]: I did both changes. Thanks.

File: lib/src/pages/misc/crop.dart (hunk 1 above)
Comment (xclud) [reply lib/src/pages/misc/crop.dart]: I've added my author information.

**Comment by lohanidamodar** [comment]
Thank you for your contribution. I have merged, and soon it will be available in the app too.

**Pull request merged**